sea-query-binder/Cargo.toml

```diff
@@ -14,7 +14,7 @@ keywords = [ "database", "sql", "mysql", "postgres", "sqlite" ]
 
 [dependencies]
 sea-query = { version = "^0.25", path = "../" }
-sqlx = { version = "^0.5.6", optional = true }
+sqlx = { version = "^0.5.10", optional = true }
```

Comment:
Why not to last sqlx version? I mien: `0.5.13`

Reply:
That's the minimum requirement. That way if some clients have only updated to sqlx 1.5.10 but not later this doesn't force an update on them. But if they have the latest, it still works.